kernel/threads-asm.s

```diff
@@ -53,3 +53,11 @@ fork_entry_point:
 	call clone_thread
 	pop_callee_save_reg
 	ret
+
+.GLOBAL unwind_from_here
```

Comment:
some explanation of what this function is and why it needs to be implemented in assembly might be useful.